scripts/train/benchmarking/collect_results.py

```diff
@@ -46,13 +43,12 @@ def parse_args():
 
 
 def get_runs(args: argparse.Namespace):
-    runs = [r for r in msdk.get_runs() if args.project in r.name]
+    runs = [r for r in msdk.get_runs(include_details=True) if args.project in r.name.split('-')[0] and r.status == msdk.RunStatus("COMPLETED")]
```

Comment:
`r.status == msdk.RunStatus("COMPLETED")`

I generally left checking if complete for filter_runs
(that way the print in filter `print(f'run {run.name} has run status {run.status}')` would let me know if a run had died or if it was still running)

I thought I'd note it.

Reply:
I see -- I always assumed that you'd know if the runs from a certain project were still running, but if you think it's a good idea to have this can add back.